src/containers/gui.jsx

```diff
@@ -132,6 +133,7 @@ const ConnectedGUI = connect(
 // ability to compose reducers.
 const WrappedGui = compose(
     ErrorBoundaryHOC('Top Level App'),
+    FontLoaderHOC,
```

Comment:
I think `FontLoaderHOC` should wrap the component `VMManagerHOC`, similar to the way redux's `connect` does:
 - The `fontsLoaded` prop that `FontLoaderHOC` provides is consumed by `VMManager `
 - `VMManager ` can't function without `fontsLoaded`
 - The font loading behavior is an aspect of project loading, more specific to `VMManager` than `GUI`

Reply:
you mean, take it out of this compose list, and put it directly into the VMManagerHOC file?

Reply:
I think he means move it down 2 lines

Reply:
I originally put it at the top because I thought it might need to be the first request completed, before we fetched; we even tried rendering null in FontLoaderHOC until fonts were loaded, but that doesn't work because (I think??) the fonts don't present themselves until you load the components that use them.

I could put it wherever, just not sure which place Ray means... "similar to the way redux's connect does" makes me think he means, like

`export FontLoaderHOC(VMManagerHOC)` 

or something...

Reply:
>> because (I think??) the fonts don't present themselves until you load the components that use them.

The fonts should be present in CSS on the page before React lifecycles start happening. I think the issue was that without the stage attached, the renderer isn't given by the GUI to the VM, and something was trying to access the renderer too soon.

Reply:
Yes I meant in `VMManagerHOC`, something like:
```jsx
return connect(
         mapStateToProps,
         mapDispatchToProps,
         mergeProps
     )(FontLoaderHOC(VMManager));
```

Although now I changed my mind, I think you should leave it since wrapping it up like that defeats the benefit of HOCs being isolated from each other so they can be tested by themselves.